exchanges/kucoin/kucoin.go

```diff
@@ -840,11 +841,8 @@ func (ku *Kucoin) PostOrder(ctx context.Context, arg *SpotOrderParam) (string, e
 	default:
 		return "", fmt.Errorf("%w %s", order.ErrTypeIsInvalid, arg.OrderType)
 	}
-	resp := struct {
-		OrderID string `json:"orderId"`
-		Error
-	}{}
-	return resp.OrderID, ku.SendAuthHTTPRequest(ctx, exchange.RestSpot, placeOrderEPL, http.MethodPost, kucoinPostOrder, &arg, &resp)
+	var resp SpotOrderResponse
```

Comment:
Curious as to why this was exported? We're still returning just the order ID rather than the struct resp